### ISSUE TITLE
Error: Controller field is private and is annotated by @FXML annotation

### ISSUE THREAD

**Issue by infinite-dev22** [Paullo612/mlfx#39]
I get this error when i run my project.
![Screenshot from 2023-06-03 08-05-18](https://github.com/Paullo612/mlfx/assets/77667007/d0c6159a-3aa3-4d20-ae45-e40373862775)

**Comment by Paullo612** [comment]
That's correct. Please change injected field visibility to public or package-private. See [this](https://github.com/Paullo612/mlfx-sample/commit/6e25b50c11672b0d0c2fd847c7e94ff1b982f79c) commit in sample application. It brings changes necessary for mlfx.

Currently, there is only Micronaut framework based DI backend implementation. Micronaut cannot inject private fields without reflections.
